DataFormats/HcalIsolatedTrack/interface/IsolatedL1TPixelTrackCandidate.h

```diff
@@ -0,0 +1,152 @@
+#ifndef HcalIsolatedTrack_IsolatedPixelTrackCandidate_h
+#define HcalIsolatedTrack_IsolatedPixelTrackCandidate_h
+/** \class reco::IsolatedPixelTrackCandidate
```

Comment:
The "L1T" is missing in the above 3 lines...

Where is the *.cc implementation file? There is one for the old class src/IsolatedL1TPixelTrackCandidate.cc,
so I would expect one for the new one as well.